README.md

```diff
@@ -114,25 +103,24 @@ code, other repos are also under active development:
   for Kong.
 - [Kong CloudFormation](https://github.com/Kong/kong-dist-cloudformation):
   Kong in a 1-click deployment for AWS EC2.
-- [Kong AWS AMI](https://aws.amazon.com/marketplace/pp/B06WP4TNKL): Kong AMI on
-  the AWS Marketplace.
+- [Kong AWS AMI](https://aws.amazon.com/marketplace/pp/B06WP4TNKL): Kong AMI on the AWS Marketplace.
 - [Kong on Microsoft Azure](https://github.com/Kong/kong-dist-azure): Run Kong
-  using Azure Resource Manager.
+  using the Azure Resource Manager.
 - [Kong on Heroku](https://github.com/heroku/heroku-kong): Deploy Kong on
   Heroku in one click.
 - [Kong and Instaclustr](https://www.instaclustr.com/solutions/managed-cassandra-for-kong/): Let
-  Instaclustr manage your Cassandra cluster.
+  Instaclustr manages your Cassandra cluster.
 - [Nightly Builds][kong-nightly-master]: Builds of the master branch available
-  every morning at about 9AM PST.
+  every morning at about 9 AM PST.
 
 You can find every supported distribution at the [official installation page](https://konghq.com/install/).
 
 ## Development
 
-If you are planning on developing on Kong, you'll need a development
+If you are planning on developing in Kong, you'll need a development
 installation. The `next` branch holds the latest unreleased source code.
 
-You can read more about writing your own plugins in the [Plugin Development
+You can read more about writing your plugins in the [Plugin Development
```

Comment:
i feel that "your plugins" makes sense if they've been mentioned before.  the first time "your own plugins" sounds better.